cmd/world/cardinal/dev.go

```diff
@@ -30,24 +33,38 @@ var StopChan = make(chan struct{})
 // Cobra Setup //
 /////////////////
 
+func init() {
+	devCmd.Flags().Bool(flagWatch, false, "Dev mode with hot reload support")
+}
+
 // devCmd runs Cardinal in development mode
 // Usage: `world cardinal dev`
 var devCmd = &cobra.Command{
 	Use:   "dev",
 	Short: "Run Cardinal in development mode",
 	Long:  `Run Cardinal in development mode`,
 	RunE: func(cmd *cobra.Command, args []string) error {
+		watch, err := cmd.Flags().GetBool(flagWatch)
+		if err != nil {
+			watch = false
+		}
```

Comment:
the case where `GetBool` returns an error, watch will be set to false, so this line is not exactly necessary

Reply:
removed